src/iop/soften.c

```diff
@@ -132,7 +132,7 @@ void process(dt_iop_module_t *self, dt_dev_pixelpipe_iop_t *piece, const void *c
 
   const float w = piece->iwidth * piece->iscale;
   const float h = piece->iheight * piece->iscale;
-  const int mrad = sqrt(w * w + h * h) * 0.01;
+  const int mrad = sqrtf(w * w + h * h) * 0.01;
```

Comment:
The constant needs to be 0.01f or there will still be a conversion to double.
For this and the other occurrences in this file, it might make sense to use
```
   = dt_fast_hypotf(w,h) * 0.01f;
```
instead.